.github/workflows/build.yml

```diff
@@ -33,11 +35,7 @@ jobs:
           load: true
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          tags: wg-access-server:testing
-          labels: |
-            org.opencontainers.image.created=${{ env.DATE }}
-            org.opencontainers.image.authors=${{ env.AUTHOR }}
-            org.opencontainers.image.source=${{ github.event.repository.clone_url }}
-            org.opencontainers.image.revision=${{ github.sha }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: $${{ steps.meta.output.labels }}
```

Comment:
The `$$` looks a bit suspicious, is it intentional?

Reply:
You're right. Gonna fix this